src/sentry/scim/endpoints/utils.py

```diff
@@ -89,19 +107,19 @@ def parse_filter_conditions(raw_filters):
         key = key.strip()
         value = value.strip()
 
-        # For USERS: Unique username should always be lowercase
+        value = value[1:-1]
```

Comment:
Wrapping value with `"` or `'` is part of the spec so this assumption is okay.

Would it make sense to throw an error if it's not?

Reply:
will address in follow up pr 👍🏼